src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -127,47 +128,50 @@ function removeGiftCard() {
 function showRemainingAmount() {
   $('#giftcard-modal').modal('hide');
   document.querySelector('#giftCardLabel').classList.add('invisible');
+  const discountsContainer = document.createElement('div');
 
-  const remainingAmountContainer = document.createElement('div');
-  const remainingAmountStart = document.createElement('div');
-  const remainingAmountEnd = document.createElement('div');
-  const cancelGiftCard = document.createElement('div');
-  const remainingAmountStartP = document.createElement('p');
-  const remainingAmountEndP = document.createElement('p');
-  const cancelGiftCardP = document.createElement('p');
-  const remainingAmountStartSpan = document.createElement('span');
-  const cancelGiftCardSpan = document.createElement('span');
-  const remainingAmountEndSpan = document.createElement('span');
+  Object.values(GIFTCARD_CONSTANTS).forEach((element) => {
```

Comment:
I'm not sure if it is a good practice to add a loop for each constant. If we suddenly would add another constant that is related to giftcards, then another container will be created here. 
I would suggest to create a function instead that will create these containers and then add in the parameters which type of container to create.